aio/aio-proxy/aio_proxy/routes.py

```diff
@@ -53,7 +51,7 @@ async def search_text_endpoint(request):
     return api_response(
         request,
         extract_function=SearchParamsBuilder.get_text_search_params,
```

Comment:
Same as previous file : I still think that extract function should not be passed as an arguments here as it is 100% linked to `search_type`

An interesting refacto : maybe add a default `extract` in `SearchParamsBuilder` that takes a search_type as an argument ? `SearchParamsBuilder.extract(search_type)`

And not sure if enum are pythonic but in js I would normalize this with enums : 
```
from enum import Enum

class SearchType(Enum):
    TEXT = 1
    GEO = 2

...
# then in SearchParamsBuilder

class SearchParamsBuilder:
	....

	def extract (request, search_type):
		if(search_type == SearchType.TEXT):
			return get_text_search_params(request)
		if(search_type == SearchType.GEO):
			return get_geo_search_params(request)
		raise ('💩')

# and in search_text_endpoint
@routes.get("/search")
async def search_text_endpoint(request):
    return api_response(request, SearchType.TEXT)
```